### PR TITLE
Added new Javascript Fundamentals Course

### PR DESCRIPTION
**Pull request by jayeshchoudhary**
## What does this PR do?
Add new Javascript Fundamentals Course in English section

### Description
Namaste JavaScript is a pure in-depth JavaScript Course released for Free on Youtube. It will cover the core concepts of JavaScript in detail and everything about how JS works behind the scenes inside the JavaScript engine.

### Why is this valuable (or not)?
This course covers the core fundamentals of JavaScript which will help anyone to clear their basics first before jumping into any JS library

### How do we know it's really free?
It is available on YouTube Playlist

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course 

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -398,6 +398,7 @@
 * [Learn how to program: JavaScript](https://www.learnhowtoprogram.com/javascript) - Epicodus Inc.
 * [Learn modern JavaScript](https://scrimba.com/learn/es6) - Beau Carnes (Scrimba) (Scrimba account *required*)
 * [learn:query](https://learnquery.infinum.co)
+* [Namaste 🙏 JavaScript | In-Depth JavaScript Fundamentals Course](https://www.youtube.com/playlist?list=PLlasXeu85E9cQ32gLCvAvr9vNaUccPVNP) - Akshay Saini (YouTube)
```
Comment (davorpa) [inline courses/free-courses-en.md:401]: Nice playlist.

It name is `Namaste Javascript`.

If an excerpt is needed, is better use `:`. Vertical bars make some bugs inside links

```suggestion
* [Namaste 🙏 JavaScript: An In-Depth JavaScript Fundamentals Course](https://www.youtube.com/playlist?list=PLlasXeu85E9cQ32gLCvAvr9vNaUccPVNP) - Akshay Saini (YouTube)
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Review by davorpa — APPROVED** [review]
Supe :rocket:. Go ahead :hugs:

**Comment by jayeshchoudhary** [comment]
@davorpa who will merge this PR ?

**Comment by davorpa** [comment]
> @davorpa, @ericallam who will merge this PR ?

At this moment I don't have write access rights. I only review, pull, issue a bit

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**